protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2356,7 +2339,7 @@ func protoPathIndex(descriptorType reflect.Type, what string) int32 {
 	}
 	path, err := strconv.Atoi(strings.Split(pbtag, ",")[1])
 	if err != nil {
-		panic(fmt.Errorf("protobuf descriptor id for %s cannot be converted to a number: %s", what, err.Error()))
+		panic(fmt.Errorf("protobuf descriptor id for %s cannot be converted to a number: %v", what, err))
```

Comment:
I also prefer the old style here. I avoid using `%v` so that we can use the warnings `go vet` produces when the wrong type is used with the wrong verb.

Reply:
%v for error is ok. And %s for error is ok too. But if error is nil verb %s text this %!s(<nil>), verb %s text this <nil>. But error is handling before

Reply:
I'm not sure I understand what you're saying, but yeah we don't need to worry about the error being `nil` here so we should be able to keep using `%s`.

Reply:
Reverted this to source